sycl/include/sycl/ext/intel/experimental/cache_control_properties.hpp

```diff
@@ -0,0 +1,243 @@
+//==--------- SYCL annotated_ptr properties for caching control ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp>
+#include <sycl/ext/oneapi/experimental/prefetch.hpp>
+#include <sycl/ext/oneapi/properties/properties.hpp>
+#include <sycl/ext/oneapi/properties/property.hpp>
+#include <sycl/ext/oneapi/properties/property_value.hpp>
+
+#include <type_traits>
+
+namespace sycl {
+inline namespace _V1 {
+namespace ext {
+namespace intel {
+namespace experimental {
+
+enum class cache_mode {
```

Comment:
Please, add a warning that definition of `cache_mode` enum must match the definition from llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp.

Reply:
OK, done.